webrtc.html

```diff
@@ -1873,28 +1800,100 @@ <h4>Set the RTCSessionDescription</h4>
+                            <p>Set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiverTransport]]</a>
+                            to <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiverTransport]]</a>
+                            and set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiverTransport]]</a>
+                            to <code>null</code>.</p>
+                          </li>
+                          <li>
+                            <p>Set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiveCodecs]]</a>
+                            to <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiveCodecs]]</a>
+                            and set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiveCodecs]]</a>
+                            to an empty list.</p>
+                          </li>
+                          <li data-tests="RTCPeerConnection-setDescription-transceiver.html">
+                            <p>If the <var>transceiver</var> was created by applying the
+                            <code><a>RTCSessionDescription</a></code> that is
+                            being rolled back, and a track has never been attached to
+                            it via <code>addTrack</code>, run the following steps:</p>
+                            <ol>
+                              <li>
+                                <p><a data-lt="stop the RTCRtpTransceiver">Stop the
+                                RTCRtpTransceiver</a> <var>transceiver</var>.</p>
+                              </li>
+                              <li>
+                                <p>Remove <var>transceiver</var> from <var>connection</var>'s
+                                <a href="#transceivers-set">set of transceivers</a>.</p>
+                              </li>
+                              <li>
+                                <p><a>Set the associated remote streams</a> given
+                                <var>transceiver</var>.<a>[[\Receiver]]</a>, <var>msids</var>
+                                as the empty list, <var>addList</var>, and <var>removeList</var>.</p>
+                              </li>
+                              <li>
+                                <p>If the <var>transceiver</var>'s <a>[[\FiredDirection]]</a>
+                                is either <code>"sendrecv"</code> or <code>"recvonly"</code>,
+                                <a>process the removal of a remote track</a> given
+                                <var>transceiver</var> and <var>muteTracks</var>.</p>
+                              </li>
+                            </ol>
+                          </li>
+                          <li>
+                            <p>Otherwise, (if the <var>transceiver</var> was not just removed)
+                            run the following steps:</p>
+                            <ol>
+                              <li>
```

Comment:
Good catch! I remember thinking about this when I was writing it, but apparently I never included it.

To avoid the whole "is the current stable stream in the local or remote description/which is the answer/etc" which I think is hard to read, I introduced a [[LastStableStateAssociatedRemoteMediaStreams]] internal slot. This also means I can handle the case the same whether we are performing a removal or not.